service/orchestrator/catalogs.go

```diff
@@ -220,23 +223,44 @@ func (svc *Service) loadCatalogs() (err error) {
 		log.Errorf("Error while saving catalog %v", err)
 	}
 
-	log.Debugf("Catalog loaded with id '%s'.", catalogs[0].GetId())
+	log.Debug("Catalogs sucessfully stored.")
 
 	return
 }
 
 func (svc *Service) loadEmbeddedCatalogs() (catalogs []*orchestrator.Catalog, err error) {
-	var b []byte
+	var (
+		b        []byte
+		fileList []string
+	)
 
-	log.Infof("Loading catalogs from %s", svc.catalogsFile)
-	b, err = f.ReadFile(svc.catalogsFile)
-	if err != nil {
-		return nil, fmt.Errorf("error while loading %s: %w", svc.catalogsFile, err)
-	}
+	// Get all filenames
+	fileList, err = readFolder(svc.catalogsFolder)
 
-	err = json.Unmarshal(b, &catalogs)
-	if err != nil {
-		return nil, fmt.Errorf("error in JSON marshal: %w", err)
+	log.Infof("Loading catalogs from files %s.", strings.Join(fileList, ", "))
+
+	// Get catalog for each file
+	for i := range fileList {
+		var catalog []orchestrator.Catalog
```

Comment:
I'm a little bit confused: `catalog` is one catalog (since it is singular) but it is a slice of `orchestrator.catalog`. And later (L262) we iterate through it to add multiple catalogs of it to `catalogs`